service/src/main/java/uk/gov/mca/beacons/api/legacybeacon/application/LegacyBeaconService.java

```diff
@@ -83,24 +78,20 @@ public List<LegacyBeacon> getBatch(int batchSize, int numberAlreadyTaken) {
       .collect(Collectors.toList());
   }
 
-  public List<LegacyBeacon> delete(
-    String hexId,
-    String email,
+  public LegacyBeacon delete(
+    LegacyBeacon legacyBeaconToDelete,
     String reasonForDeletion
   ) {
     LocalDateTime todaysDate = LocalDateTime
       .now()
       .truncatedTo(ChronoUnit.SECONDS);
-    OffsetDateTime todaysOffsetDate = todaysDate.atOffset(ZoneOffset.UTC);
-
-    List<LegacyBeacon> legacyBeacons = legacyBeaconRepository.findByHexIdAndOwnerEmail(
-      hexId,
-      email
+    OffsetDateTime todaysOffsetDate = todaysDate.atOffset(
+      ZoneOffset.ofTotalSeconds(1)
```

Comment:
Is there a reason for doing this rather than just getting the time now?
> OffsetDateTime.now()

Reply:
We're doing it in a weird way because for some reason the format of the date from doing OffsetDateTime.now() wasn't in line with the format in production. See commit 15764c3a